### PR TITLE
Defaulted network name to homestead

### PR DESCRIPTION
**Pull request by asselstine**

### CODE REVIEW

File: src/apollo/client-state/web3Resolvers.js [hunk 1]
```diff
@@ -20,14 +20,15 @@ export const web3Resolvers = {
           let provider
           try {
             provider = await getWriteProvider()
+          } catch (e) {}
+          if (!provider) { return null }
```
Comment (chuckbergeron) [inline src/apollo/client-state/web3Resolvers.js:24]: Indentation may be wrong here?

File: src/web3/getNetworkName.js [hunk 2]
```diff
@@ -16,9 +16,11 @@ export async function getNetworkName () {
     networkName = network.name
   } else {
     if (!process.env.REACT_APP_DEFAULT_NETWORK_NAME) {
-      throw new Error('You must define the enviroment variable REACT_APP_DEFAULT_NETWORK_NAME')
+      networkName = 'homestead'
```
Comment (chuckbergeron) [inline src/web3/getNetworkName.js:19]: `homestead` is a rather weird word to throw in here. That's an ethers thing?

**Review by chuckbergeron — APPROVED** [review]

File: src/web3/getNetworkName.js (hunk 2 above)
Comment (asselstine) [reply src/web3/getNetworkName.js:19]: Yeah it's an Ethers.js thing.  However, "homestead" is actually the official name of the first network.

File: src/apollo/client-state/web3Resolvers.js (hunk 1 above)
Comment (asselstine) [reply src/apollo/client-state/web3Resolvers.js:24]: Indentation is right; it's just janky code.  We need the empty catch at the moment.

**Pull request merged**